addons/killtracker/XEH_postInit.sqf

```diff
@@ -35,20 +35,23 @@ INFO("Running Kill Tracking");
 // Variables:
 GVAR(eventsArray) = [];
 GVAR(outputText) = "None";
```

Comment:
Maybe change this line to `Total Kills: 0` so it's consistant

Reply:
Sure thing. I didn't even realize it would show None if you did nothing.